remote_search.py

```diff
@@ -81,12 +82,13 @@ def __post_init__(self):
             AUDIO_FIELD_NAME: RemoteMediaInfo(AUDIO_FIELD_NAME, self.sound_url, MediaType.sound),
         }
         self._mapping = {
-            "SentKanji": self.sent_kanji,
-            "SentFurigana": self.sent_furigana,
-            "SentEng": self.sent_eng,
+            "Expression": self.sent_kanji,
+            "Reading": self.sent_furigana,
+            "English": self.sent_eng,
             AUDIO_FIELD_NAME: self.audio.as_anki_ref(),
             IMAGE_FIELD_NAME: self.image.as_anki_ref(),
             "Notes": self.notes,
```

Comment:
Originally, the fields were named to match our official note type (Japanese sentences). If we change the field names, we will also have to change the official note type, and the users will have to change (or update) their settings as well. 

I we want to address the situation when people don't use the official note type, we should let the users assign field names in Settings.